docs/website/package-lock.json

```diff
@@ -1,11114 +0,0 @@
-{
```

Comment:
Any reason for deleting this file?

Reply:
ah, that was a mistake :-/ I'll remove this from the commit

Reply:
fixed